frontend/src/components/dashboard/Dashboard.tsx

```diff
@@ -7,20 +7,20 @@ import QuestionStatisticsCard from "./QuestionStatisticsCard";
 
 const Dashboard = () => {
   return (
-    <div className="grid grid-rows-2 grid-cols-4 gap-4 p-[30px] h-[92vh]">
+    <div className="grid md:grid-rows-2 md:grid-cols-4 gap-4 p-[30px] sm:grid-row-3 sm:grid-cols-2 container mx-auto">
```

Comment:
![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g05/assets/69421979/a548bcdf-fce5-4dde-ad41-38fecc2f27d1)
Looks like without the `h-[92vh]`, the dashboard looks a bit too long in terms of vertical (not sure what is the right word to describe it haha). I think we should put this height limit if it does not cause any problem?

Reply:
I don't think the problem is just the height of the image? As all the panels also gets stretched.